File: Chitunion/Web(code refactoring)/XYAuto.ChiTu2018/XYAuto.ChiTu2018.Entities/Chitunion2017/Title_Media_Mapping.cs
namespace XYAuto.ChiTu2018.Entities.Chitunion2017
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Title_Media_Mapping
    {
        [Key]
        public int RecID { get; set; }

        public int TitleID { get; set; }

        public int MediaType { get; set; }

        public int MediaID { get; set; }

        public int? Status { get; set; }

        public DateTime? CreateTime { get; set; }

        public int? CreateUserID { get; set; }
    }
}

File: Chitunion/Web(java)/src/main/java/com/xyauto/domain/TitleMediaMapping.java
package com.xyauto.domain;

import java.util.Date;

import lombok.Data;

@Data
public class TitleMediaMapping {
	private Integer recId;
	private Integer titleId;
	private Integer mediaType;
	private Integer mediaId;
	private Integer status;
	private Date createTime;
	private Integer createUserId;
}
